packages/billboard-widget/src/hooks/useAuth.ts

```diff
@@ -88,7 +88,7 @@ export const useAuth = (
             wallet.address,
         );
 
-        const ensName = `${clientProps.siweAddress}.user.ethprague.dm3.eth`;
+        const ensName = `${clientProps.siweAddress}.bb-user.dm3.eth`;
```

Comment:
It think in one of the branches we had for istanbul this was already part of the client props obj